src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,6 +5,9 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String type;
+    private String weight;
```

Comment:
Consider changing the data type of the `weight` field from `String` to a more appropriate numeric type, such as `double` or `int`, depending on the precision required. This will allow for more accurate calculations and comparisons if needed.